releng/org.eclipse.xtext.sdk.feature/feature.xml

```diff
@@ -74,6 +74,34 @@ SPDX-License-Identifier: EPL-2.0
       <import feature="org.eclipse.emf.mwe2.language.sdk" version="2.12.2" match="greaterOrEqual"/>
    </requires>
 
+   <plugin
+         id="org.eclipse.xtext.xtext.generator"
+         download-size="0"
+         install-size="0"
+         version="0.0.0"
+         unpack="false"/>
+         
+   <plugin
+         id="org.eclipse.xtext.xtext.generator.source"
+         download-size="0"
+         install-size="0"
+         version="0.0.0"
+         unpack="false"/>
+
+    <plugin
```

Comment:
should this go to xtext.xtext.ui?

Reply:
Yes, they should. The SDK should add the examples and otherwise be a compilation of other features.

Reply:
Hmm purexbase ... I doubt that this should be part of the SDK even.

Reply:
sdk seems "everything". moved generator and wizard to xtext.xtext.ui